src/codechecker_script.py

```diff
@@ -280,13 +280,12 @@ def resolve_symlinks():
     files_processed = 0
     for root, _, files in os.walk(analyze_outdir):
         for filename in files:
-            if re.search("clang-tidy", filename):
```

Comment:
To be honest I dont remember why this condition was added.
I'm curious about all possible positive and especially negative scenarios.
In other words, are we sure that we should process all plist and yaml files?